docs/content/commands/rclone_hashsum.md

```diff
@@ -16,6 +16,8 @@ Produces a hash file for all the objects in the path using the hash
 named.  The output is in the same format as the standard
 md5sum/sha1sum tool.
 
+By default, the hash is requested from the remote.  If the hash is not supported, no hash will be returned.
```

Comment:
This text needs to go in the help for `rclone hashsum`

I run `make backenddocs` as part of the release, so I prefer commits without patches to the auto generated docs.

Anything in docs/contents/commands/ is auto generated